### ISSUE TITLE
tool snippet gives linter errors

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy-language-server#91]
More an planemo / galaxy issue but I guess its good to keep track here:

```
Applying linter xml_order... CHECK
.. INFO: Unknown tag [xrefs] encountered, this may result in a warning in the future.
.. INFO: Unknown tag [edam_topics] encountered, this may result in a warning in the future.
.. INFO: Unknown tag [edam_operations] encountered, this may result in a warning in the future.

....

.. ERROR: Invalid XML found in file: mztab2tsv.xml. Errors [/tmp/tmp48f2aqmq:1:0:ERROR:SCHEMASV:SCHEMAV_CVC_COMPLEX_TYPE_3_2_1: Element 'tool', attribute 'license': The attribute 'license' is not allowed.
```

**Comment by bgruening** [comment]
@bernt-matthias if you install the latest (dev) version of galaxy-tool-util https://pypi.org/project/galaxy-tool-util/ that should be fixed.

**Comment by davelopez** [comment]
Yes, since the [v0.3.1 release](https://github.com/galaxyproject/galaxy-language-server/releases/tag/v0.3.1) of the VSCode/Codium extension we are using the [galaxy-tool-util==21.1.0.dev4](https://github.com/galaxyproject/galaxy-language-server/blob/b88afafba09acb36263820c7133f6d5f9a0368f3/server/requirements.txt#L4) wich should be the same for the latest release of planemo [here](https://github.com/galaxyproject/planemo/blob/bea6a16ab65e1925485cdd0f109434703e639368/requirements.txt#L10)

**Comment by davelopez** [comment]
After further investigation on this issue, the problem seems to be a mismatch between the [latest IUC Coding Style documentation](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#coding-style) and the latest [`TAG_ORDER`](https://github.com/galaxyproject/galaxy/blob/90b83bc83859bdb221cbc32a4098aad355a8242b/lib/galaxy/tool_util/linters/xml_order.py#L8) definition in galaxy-tool-util.
There is also an open discussion and a draft PR (https://github.com/galaxy-iuc/standards/pull/58/files) that may potentially change the current order, so maybe is better to wait until it is resolved. I'll update the snippet to reflect the final decision when it is made.

Thank you @bernt-matthias for bringing that up!
